### ISSUE TITLE
WCAG2.2 - Audit React components

### ISSUE THREAD

**Issue by ariellalgilmore** [carbon-design-system/carbon#18875]
The overarching goal of this audit is to determine what needs to be fixed in `@carbon/react` to fully conform to WCAG 2.2

1. Take the [audit](https://ibm.box.com/s/e2hhcmw1u6teofxgqx2l1nlw89mbxii0) prepared in 2024 by Michael Gower and open new issues for anything that needs addressed. These issues should be added as sub-issues of https://github.com/carbon-design-system/carbon/issues/15592
2. Find any existing issues and add them as sub-issues to https://github.com/carbon-design-system/carbon/issues/15592

Use this as a guide for evaluating: https://medium.com/design-ibm/assessing-components-and-patterns-for-wcag-2-2-47ee3f0f468e

**Comment by warrenmblood** [comment]
Updated WCAG 2.2 Audit: [Box link](https://ibm.box.com/s/c34pbo9ra1imi6lpcsnl3c5smssop16a)
